### PR TITLE
Editorial review on backup and recovery doc

### PR DESCRIPTION
**Pull request by g-murray**
Performing an editorial review of entire doc ahead of 2.5 EA.

Doc title : aap-operator-backup

### CODE REVIEW

File: downstream/modules/platform/proc-aap-hub-restore.adoc [hunk 1]
```diff
@@ -12,8 +12,8 @@ The name specified for the new AutomationHub custom resource must not match an e
 
 .Prerequisites
 
-* You must be authenticated with an Openshift cluster.
-* The {HubName} has been deployed to the cluster.
+* You must be authenticated with an OpenShift cluster.
+* You have deploed {HubName} on the cluster.
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-aap-hub-restore.adoc:16]: Think this is a typo `deploed` should be `deployed`?

**Review by michellemacrh — COMMENTED** [review]
Just a teeny tiny typo - otherwise LGTM

File: downstream/modules/platform/proc-aap-hub-restore.adoc (hunk 1 above)
Comment (g-murray) [reply downstream/modules/platform/proc-aap-hub-restore.adoc]: Good spot! Fixed!

**Comment by michellemacrh** [comment]
LGTM

**Pull request merged**